src/test/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorServiceSpec.scala

```diff
@@ -206,6 +206,104 @@ class PolicyEvaluatorServiceSpec extends FlatSpec with Matchers with TestSupport
     res.unsafeRunSync()
   }
 
+
+  "hasPermission" should "return false if given action is not allowed for a user using the short circuit" in {
+    val user = genUserInfo.sample.get
```

Comment:
this is fine as is...but you can also try using `forAll`, so that the test will generate a few samples instead of just one, something like https://github.com/broadinstitute/sam/blob/842caafa451d7100407a5023ed57c668d5378c2a/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/StandardUserInfoDirectivesSpec.scala#L32

Reply:
I was just following the convention of the rest of the tests, but this is good to know about, thanks